lib/core/Head.js

```diff
@@ -134,13 +134,23 @@ class Head extends React.Component {
 
         {/* External resources */}
         {this.props.config.stylesheets &&
-          this.props.config.stylesheets.map(source => (
-            <link rel="stylesheet" key={source} href={source} />
-          ))}
+          this.props.config.stylesheets.map(source => {
+            if (source instanceof Object) {
+              return source.href ? (
+                <link rel="stylesheet" key={source.href} {...source} />
+              ) : null;
+            }
+            return <link rel="stylesheet" key={source} href={source} />;
+          })}
         {this.props.config.scripts &&
-          this.props.config.scripts.map(source => (
-            <script type="text/javascript" key={source} src={source} />
-          ))}
+          this.props.config.scripts.map(source => {
```

Comment:
Better written as:

```jsx
this.props.config.scripts.map(script => {
  const scriptProps = Object.assign({}, 
    { type: 'text/javasript' }, 
    script instanceOf 'Object' ? script : { src: source },
  );
  return <script {...source} />;
});
```

Reply:
We need to include the key, 
``` js
this.props.config.scripts && 
  this.props.config.scripts.map(source => {
    const scriptProps = Object.assign(
        {type: 'text/javasript'},
        source.src ? source : {src: source} // include key in this line is more complexity than use ?..: ...
    );
    return <script {...scriptProps} />;
})
```

I guess if we don't need key, we can do as your example.